python/pyspark/sql/functions.py

```diff
@@ -1028,20 +1028,29 @@ def to_timestamp(col, format=None):
 
 
 @since(1.5)
-def trunc(date, format):
+def trunc(data, truncParam):
```

Comment:
@wangyum, would you mind revert this renaming? This breaks the compatibility if user script calls this by

```python
trunc(..., format= ...)
trunc(date=..., format= ...)
```